src/main/assets/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0'
+        classpath 'com.android.tools.build:gradle:3.0.1'
```

Comment:
Problem with customizing AGP version it that it is hardcoded here.

Reply:
is this file used? 
if I set androidGradlePluginVersion to 3.0.0 I have only one reference

$ egrep -ir 'com.android.tools.build:gradle' modules/gennyCLI
./build.gradle:        classpath 'com.android.tools.build:gradle:3.0.0'

Reply:
This file was copied directly to the project folder, but after https://github.com/borisf/android-studio-poet/commit/b8d53687029d53e76dbb49801fcd7f440d54bbec#diff-469f26b37acf957d7e037aa3fc78cd01 this file is no longer used, I should have removed it on the same commit.

Reply:
I removed it